app/src/main/java/net/bible/android/view/activity/workspaces/WorkspaceSelectorActivity.kt

```diff
@@ -465,7 +465,7 @@ class WorkspaceSelectorActivity: ActivityBase() {
         val menu = popup.menu
         popup.menuInflater.inflate(R.menu.workspace_popup_menu, menu)
         val delItem = menu.findItem(R.id.deleteWorkspace)
-        delItem.isEnabled = workspaceEntity.id != windowControl.windowRepository.id
+        delItem.isEnabled = dataSet.size != 1 // cannot delete the only workspace
```

Comment:
This should always work, but it might be better practice to use "> 1" instead?

Reply:
```suggestion
        delItem.isEnabled = dataSet.size > 1 // cannot delete the only workspace
```